react-form-controls/app-name/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormEvent, useState } from 'react';
+
+export function RegistrationFormControlled() {
+  const [userName, setUserName] = useState('');
+  const [passWord, setPassWord] = useState<string>();
+
+  function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    const form = new FormData(event.currentTarget);
```

Comment:
This one is controlled, so you don't need to use FormData to get the values. The values are already in your `userName` and `passWord` state variables. So you could just log those.